dask/dataframe/tests/test_dataframe.py

```diff
@@ -5770,3 +5770,20 @@ def test_to_backend():
         # Moving to a "missing" backend should raise an error
         with pytest.raises(ValueError, match="No backend dispatch registered"):
             df.to_backend("missing")
+
+
+@pytest.mark.parametrize("func", ["first", "last", "max", "sum"])
+def test_transform_getitem_works(func):
+    df = pd.DataFrame({"ints": [1, 2, 3], "grouper": [0, 1, 0]})
+
+    ddf = dd.from_pandas(df, npartitions=2)
+    meta = df.groupby("grouper").transform(func)
```

Comment:
Can you add a comment here that explains that we are testing the broadcasting behavior? Technically, none of these functions are proper transformers, they all aggregate and the broadcast back to the original shape. 

cc @rhshadrach just checking that this is something that we intend to keep in pandas?

Reply:
Yes, broadcasting a reduction so that it is a transform is very useful for perf, e.g. computing zscores.